src/util.cpp

```diff
@@ -349,7 +349,7 @@ int LogPrintStr(const std::string &str)
         ret = fwrite(strTimestamped.data(), 1, strTimestamped.size(), stdout);
         fflush(stdout);
     }
-    else if (fPrintToDebugLog)
+    if (fPrintToDebugLog)
```

Comment:
I'm fine with making these non-exclusive, however we still need a way to disable logging to `debug.log`. Some logging setups (e.g. when using systemd) process the `-printtoconsole` output, and don't need a duplicate logging to disk.
Maybe `-nodebuglogfile`?

Reply:
Maybe `-debuglogfile=0` as a special case?

Reply:
What about `-debuglogfile=/dev/null` as a special case .-)

If we are to add a separate option, then perhaps something starting with `-disable` to make it consistent with `-disablewallet`? We don't have any `bitcoind` command-line options starting with `-no` AFAIK.

Also `-nodebuglogfile` can be parsed as "node bug logfile" :-)

Reply:
Actually -noX is interpreted by the args manager as -X=0 implicitly anyway, converted by the InterpretNegativeSetting() function. So @laanwj and @Sjors suggestions are equivalent :)